package.json

```diff
@@ -29,47 +29,48 @@
     "@sentry/hub": "7.119.1",
     "@sentry/types": "7.119.1",
     "@sentry/utils": "7.119.1",
-    "@sentry/wizard": "^3.32.0"
+    "@sentry/wizard": "^3.34.2"
   },
   "devDependencies": {
+    "@jest/types": "^26.0.0",
+    "@rollup/plugin-commonjs": "^28.0.1",
+    "@rollup/plugin-node-resolve": "^15.3.0",
+    "@rollup/plugin-terser": "^0.4.4",
+    "@rollup/plugin-typescript": "^12.1.1",
     "@sentry-internal/eslint-config-sdk": "7.119.1",
     "@sentry-internal/typescript": "7.119.1",
     "@types/cordova": "0.0.34",
-    "@types/jest": "^24.0.18",
+    "@types/jest": "^26.0.24",
     "@types/node": "^12.7.5",
-    "codecov": "^3.5.0",
-    "eslint": "7.6.0",
-    "jest": "^24.9.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^1.18.2",
+    "@types/prettier": "<2.6.0",
+    "eslint": "^7.13.0",
+    "jest": "^26.6.3",
+    "json-schema": "^0.4.0",
+    "prettier": "^3.3.3",
     "prettier-check": "^2.0.0",
-    "replace-in-file": "^4.1.3",
-    "rimraf": "^3.0.0",
+    "replace-in-file": "^8.2.0",
     "rollup": "^2.79.2",
-    "rollup-plugin-commonjs": "^9.3.4",
-    "rollup-plugin-node-resolve": "^4.2.3",
-    "rollup-plugin-terser": "^4.0.4",
-    "rollup-plugin-typescript2": "^0.21.0",
-    "ts-jest": "^24.1.0",
-    "typescript": "^3.6.3"
+    "ts-jest": "^26.5.0",
+    "typescript": "^3.6.3",
+    "yarn": "^1.22.22"
   },
   "scripts": {
     "build:bundle": "rollup --config",
     "build:typescript": "tsc -p tsconfig.build.json",
-    "build": "run-s clean build:typescript build:bundle",
```

Comment:
npm-run-all was replaced buy https://www.npmjs.com/package/npm-run-all2 for of the original